code/modules/mob/mob_defines.dm

```diff
@@ -198,3 +198,6 @@
 	var/list/progressbars = null	//for stacking do_after bars
 
 	var/list/tkgrabbed_objects = list() // Assoc list of items to TK grabs
+
+	var/list/emote_list = list() // Initial emotes
```

Comment:
One thing that's kind of annoying about this is that each mob maintains its own list of emotes. Once byond gets proper static variables mobs can save memory by sharing lists of emotes. Couldn't go tg's route of having a global set of emotes, since that would mean snowflaking everywhere given how Paradise's emotes are so varied between different things...not sure if there's a better way to do this?

Reply:
You could have a static list that is shared, then have something like

`var/list/special_snowflake_emote_list = list()` and special emotes are stored there, that are unique to a species/mob/etc.

Reply:
Problem with that is Paradise's emotes are so snowflaked/sophisticated that usually each type of mob needs its own set of emotes (and stuff like the set of robot emotes are shared across multiple types of mobs).

I've got an idea that might work: a static associative list of all sets of emotes, with keys that associate to each set of emotes (eg "living", "synth", "human"), and then each mob can just have a list of strings that point to the relevant sets of emotes that they want.

Reply:
I don't really understand what you mean by "Paradise's emotes are so varied that I had to change this" -- Why can't you implement istype checks in the emotes themselves?